### DISCUSSION TITLE
Why `make emscripten` is removed?

### DISCUSSION THREAD

**Discussion by glyh** [janet-lang/janet#946] [General]
Seems like it's removed here: https://github.com/janet-lang/janet/commit/52dd0f132a6805c7e198bab360ea51c345dd21db

I personally think it would be handy if there's such one command stuff that make it just work.

**Comment by bakpakin** [comment]
It was removed because Janet's build process has changed, and the best way to build with emscripten is to use janet.c and janet.h and build yourself. Also, there are a few ways to build with emscripten that depend on how you want to embed in your website.
